modules/ROOT/pages/tokenization-example.adoc

```diff
@@ -1,41 +1,47 @@
-= Configure and Use Tokenization
+= Expose Tokenization as a Service Directly to Mule Applications
 
-== Set up Tokenization with API Gateway
+In some cases, for example, if a lot of complex data processing needs to take place before traffic gets to the tokenization/de-tokenization service, it makes sense to set up a Mule app to which Edge can route the traffic. In this scenario, the Mule app allows the flexibility to process the data and then send the traffic to the tokenization service. 
```

Comment:
Let's not put the example in the middle of the sentence telling us about some cases.